source/extensions/filters/http/ext_proc/ext_proc.cc

```diff
@@ -221,7 +221,8 @@ void Filter::onDestroy() {
 }
 
 FilterHeadersStatus Filter::onHeaders(ProcessorState& state,
-                                      Http::RequestOrResponseHeaderMap& headers, bool end_stream) {
+                                      Http::RequestOrResponseHeaderMap& headers, bool end_stream,
+                                      ProtobufWkt::Struct* proto) {
```

Comment:
This should be pass by reference.

Reply:
We use `nullptr` to indicate that attributes are not configured. This is on Line 243